openmetadata-service/src/test/java/org/openmetadata/service/resources/dqtests/TestCaseResourceTest.java

```diff
@@ -771,11 +771,12 @@ void test_getSimplelistFromSearch(TestInfo testInfo) throws IOException, ParseEx
               .withTimestamp(TestUtils.dateToTimestamp(String.format("2021-09-%02d", i)));
       putTestCaseResult(testCase.getFullyQualifiedName(), testCaseResult, ADMIN_AUTH_HEADERS);
     }
+    TestCase testCaseForEL = testCases.get(0);
```

Comment:
explicitly got the first test case created in this test method vs relying on what is listed as other test cases are created and will appear in the list when the test class is ran